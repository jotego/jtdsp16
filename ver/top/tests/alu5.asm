x=2
y=0x63
p=x*y
a0=p
a1=a0+p
a0=a0-p
end:
goto end
