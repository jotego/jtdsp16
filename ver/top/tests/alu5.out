r0=0x0000
r1=0x0000
r2=0x0000
r3=0x0000
rb=0x0000
re=0x0000
j =0x0000
k =0x0000
pr=0x0000
pt=0x0000
i =0x0000
a0=0x000000000
a1=0x00000018c
x=0x0002
y=0x00630000
p=0x000000c6
c0=0x0000
c1=0x0000
c2=0x0000
auc=0x0000
psw=0x4000
